clio-server/config/site.conf

```diff
@@ -0,0 +1,163 @@
+ServerAdmin ${SERVER_ADMIN}
+ServerName ${SERVER_NAME}
+ServerTokens ProductOnly
+TraceEnable off
+
+LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+LogLevel ${LOG_LEVEL}
+
+Header unset X-Frame-Options
+Header always set X-Frame-Options SAMEORIGIN
+
+ProxyTimeout ${PROXY_TIMEOUT}
+
+LDAPCacheTTL ${LDAP_CACHE_TTL}
+
+<VirtualHost _default_:${HTTPD_PORT}>
+ErrorLog /dev/stdout
```

Comment:
I don't know anything about this config file, but if possible it would be easier to read if the xml-like element contents were indented.